lib/worker_wrapper.js

```diff
@@ -381,6 +382,10 @@ class WorkerWrapper extends EventEmitterEx {
         }
 
         setImmediate(() => {
+            // this call of setup sets inspectPort for node js debugger
+            // it should be here so that every worker can receive the same debugger port after restarting
+            this._master.setup({inspectPort: this.options.inspectPort + this.wid})
```

Comment:
This calculation should be made when creating WorkerWrapper.

Reply:
Move it in this._workerInspectPort variable